extensions/resteasy-reactive/quarkus-resteasy-reactive-links/deployment/src/test/java/io/quarkus/resteasy/reactive/links/deployment/TestResource.java

```diff
@@ -68,4 +68,16 @@ public TestRecord getBySlug(@PathParam("slug") String slug) {
                 .findFirst()
                 .orElseThrow(NotFoundException::new);
     }
+
+    @GET
+    @Path("/slugOrId/{slugOrId}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @RestLink(rel = "get-by-slug-or-id", entityType = TestRecord.class)
```

Comment:
Not sure if this should test setting the `entityType` or that it should work by default, given the return type.

Reply:
I've removed the `entityType` field as it was not necessary.